format/Message.fbs

```diff
@@ -0,0 +1,113 @@
+namespace arrow.flatbuf;
+
+table Tuple {
```

Comment:
`Struct` is a reserved word in flatbuffers, so we used `Tuple` to disambiguate. I will add a comment